File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.py
# Last updated: 8/2/2025, 4:05:20 PM
class Solution(object):
    def removeDuplicates(self, nums):
        nums[:]=sorted(set(nums))
        return len(nums)
        """
        :type nums: List[int]
        :rtype: int
        """
        

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 8/2/2025, 4:05:16 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        int c=1;
        for(int i=1;i<nums.length;i++){
            if(nums[i]!=nums[i-1]){
                nums[c]=nums[i];
                c++;
            }
        }
        return c;
    }
}